commands/deploy/deploy.go

```diff
@@ -36,6 +37,7 @@ func Setup(middlewares []middleware.Middleware) *cobra.Command {
 		Long:    "Deploy your site",
 	}
 	ccmd.Flags().StringVarP(&cmd.base, "base-directory", "b", "", "directory to publish")
+	ccmd.Flags().StringVarP(&cmd.publish, "publish-directory", "b", "", "directory to publish")
```

Comment:
I don't think you can have two with the same short char

Reply:
fixed